src/xbps/repositories/mirrors/index.md

```diff
@@ -13,12 +13,12 @@ packages available.
 By default XBPS will reach out to <https://repo-default.voidlinux.org> which may
 map to any tier 1 mirror.
 
-| Repository                                     | Location         |
-|------------------------------------------------|------------------|
-| <https://repo-fi.voidlinux.org/>               | EU: Finland      |
-| <https://mirrors.servercentral.com/voidlinux/> | USA: Chicago     |
-| <https://repo-us.voidlinux.org/>               | USA: Kansas City |
-| <https://repo-de.voidlinux.org/>               | EU: Germany      |
+| Repository                                     | Location        |
+|------------------------------------------------|-----------------|
+| <https://repo-de.voidlinux.org/>               | DE: Münster     |
```

Comment:
this is an example of what I was saying about geoip. That mirror is not in Münster, it's in Frankfurt

Reply:
Ok. I will remove the cities which were not explicit in the previous version. In this case, shall we keep just the country code? Like this `DE:` ?

Reply:
Mey be something like `CN: ?` (used Chine just as an example of missing city).

Reply:
it might be better to have a separate region column, for e.g. Asia/Europe/North America/South America/Africa/Oceania, then the location column could be `country` or `city, country`

Reply:
See if it is OK now. :-)